### DISCUSSION TITLE
Laravel 7 error when try to install

### DISCUSSION THREAD

**Discussion by CarlesMR** [rappasoft/laravel-livewire-tables#421] [Q&A]
I'm trying to install the package in a Laravel with version 7, but return this error.

![imagen](https://user-images.githubusercontent.com/50807606/127141849-68ab0ec1-6b32-440c-aa97-f97122ff6cda.png)

Its a fresh install i dont know why is giving this error.

Any idea?

**Comment by rappasoft** [comment]
Says it in the error. It wants Laravel 8 but you only have Laravel 7.30.4

https://github.com/rappasoft/laravel-livewire-tables/blob/master/composer.json#L24

**Reply by CarlesMR** [reply]
Okey, so i understand its only usefull for Laravel 8?

**Reply by rappasoft** [reply]
Yes currently, I haven't had time to see if it works with Laravel 7.
